androidapp/Connexus/app/src/main/java/com/green/apt/connexus/ImageAdapter.java

```diff
@@ -0,0 +1,89 @@
+package com.green.apt.connexus;
+
+/**
+ * Created by rayolanderos on 10/18/17.
+ */
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.GridView;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.io.InputStream;
+import java.util.List;
+
+public class ImageAdapter extends BaseAdapter{
+    private Context mContext;
+    private List<String> imageUrls;
+
+    public ImageAdapter(Context c, List<String> imageUrls) {
+        mContext = c;
+        this.imageUrls = imageUrls;
+    }
+
+    public int getCount() {
+        return imageUrls.size();
+    }
+
+    public Object getItem(int position) {
+        return null;
```

Comment:
I think is good practice to return `imageUrls.get(position);` here.

Reply:
Not sure what you mean here. I grabbed this code from the android tutorials but the imageUrls are just an element in the grid. Granted, in our grid, there is only image elements inside but the adapter for the grid can include other things, not just the images.